logfire/_internal/config.py

```diff
@@ -889,7 +901,6 @@ def get_tracer_provider(self) -> ProxyTracerProvider:
         Returns:
             The tracer provider.
         """
-        self.warn_if_not_initialized('No logs or spans will be created')
```

Comment:
I moved these warnings to prevent `logfire.instrument_fastapi` emitting similar warnings 3 times.